editor/plugins/shader_editor_plugin.cpp

```diff
@@ -405,11 +411,13 @@ void ShaderEditorPlugin::drop_data_fw(const Point2 &p_point, const Variant &p_da
 
 		for (int i = 0; i < files.size(); i++) {
 			String file = files[i];
-			if (!ResourceLoader::exists(file, "Shader")) {
-				continue;
+			Ref<Resource> res;
+			if (ResourceLoader::exists(file, "Shader")) {
+				res = ResourceLoader::load(file);
+			}
+			if (ResourceLoader::exists(file, "ShaderInclude")) {
+				res = ResourceLoader::load(file);
 			}
```

Comment:
```suggestion
			if (ResourceLoader::exists(file, "Shader") || ResourceLoader::exists(file, "ShaderInclude")) {
				res = ResourceLoader::load(file);
			}
```

Reply:
When you apply the suggestion, make sure to squash the commits. We prefer all PRs to contain only a single commit in them. You can keep the current commit message as the final message.